app/views/bus_stops/by_sequence.haml

```diff
@@ -1,13 +1,16 @@
 .page-title Route #{@route.number} Bus Stops
 
-.container
+%div
+  = link_to 'View by status', by_status_bus_stops_path(number: @route.number)
+- if @collection.first.first.nil?
```

Comment:
Could you explain `.first.first` to me?

Reply:
He wanted to check that `stops` wasn't empty. Probably better to put `@stops = @route.bus_stops` in the controller and rewrite the if?

Reply:
Hm. I talked to @jforgue and it seems like I'm wrong. I'll let him explain.